flaml/automl.py

```diff
@@ -2200,7 +2200,7 @@ def custom_metric(
                 }
             }
         }
-            fit_kwargs_by_estimator: dict, default=None | The user specified keywords arguments, grouped by estimator name.
+        ```fit_kwargs_by_estimator: dict, default=None | The user specified keywords arguments, grouped by estimator name.
```

Comment:
```suggestion
        ```

        fit_kwargs_by_estimator: dict, default=None | The user specified keywords arguments, grouped by estimator name.
```